angular.json

```diff
@@ -263,22 +258,28 @@
         "build": {
           "builder": "@angular-devkit/build-ng-packagr:build",
           "options": {
-            "project": "src/clr-core/ng-package.json",
-            "tsConfig": "src/clr-core/tsconfig.json"
+            "tsConfig": "src/clr-core/tsconfig.lib.json",
+            "project": "src/clr-core/ng-package.json"
+          },
+          "configurations": {
+            "production": {
+              "tsConfig": "src/clr-core/tsconfig.lib.prod.json"
+            }
           }
         },
         "test": {
           "builder": "@angular-devkit/build-angular:karma",
           "options": {
             "main": "src/clr-core/test.ts",
             "tsConfig": "src/clr-core/tsconfig.spec.json",
-            "karmaConfig": "src/clr-core/karma.conf.js"
+            "karmaConfig": "src/clr-core/karma.conf.js",
+            "codeCoverageExclude": ["src/clr-core/common/css-vars/css-vars-ponyfill-fork.ts"]
```

Comment:
nit: this file no longer exists so no need to add to excludes

Reply:
Fixed.